### DISCUSSION TITLE
error for autocore for build ax3600

### DISCUSSION THREAD

**Discussion by Tony91590** [Boos4721/openwrt#143] [Bugs/问题反馈]
hello in the system log I regularly have the following error I think this is the autocotre

Tue May 31 21:37:05 2022 daemon.err uhttpd[2750]: awk: cmd. line:1: Network is down
Tue May 31 21:37:05 2022 daemon.err uhttpd[2750]: awk: /stats/cpu_load_ubi: No such file or directory

Good day thank Tony 
thank you again for your great work

**Comment by Boos4721** [comment]
**NSS usage** has been dropped
so `Tue May 31 21:37:05 2022 daemon.err uhttpd[2750]: awk: /stats/cpu_load_ubi: No such file or directory`
